src/file-uploader/file-uploader.module.ts

```diff
@@ -5,15 +5,15 @@ import { CheckmarkFilled16Module } from "@carbon/icons-angular/lib/checkmark--fi
 import { WarningFilled16Module } from "@carbon/icons-angular/lib/warning--filled/16";
 
 import { FileUploader } from "./file-uploader.component";
-import { File } from "./file.component";
+import { FileComponent } from "./file.component";
 import { ButtonModule } from "../button/button.module";
 import { LoadingModule } from "../loading/loading.module";
 
 export { FileUploader } from "./file-uploader.component";
 
 @NgModule({
-	declarations: [FileUploader, File],
-	exports: [FileUploader],
+	declarations: [FileUploader, FileComponent, File],
+	exports: [FileUploader, FileComponent, File],
```

Comment:
Sorry, to be more specific these should be:
```ts
    declarations: [FileUploader, FileComponent],
	exports: [FileUploader, FileComponent],
```

And there should be an `export { FileComponent, File } from "./file.component";` next to the `FileUploader` export :+1: